pkg/network/ovn_kubernetes.go

```diff
@@ -1460,9 +1460,7 @@ func isIPsecMachineConfigActive(infra bootstrap.InfraStatus) bool {
 	}
 	ipSecPluginOnMasterNodes := hasSourceInMachineConfigStatus(infra.MasterMCPStatus, infra.MasterIPsecMachineConfig.Name)
 	ipSecPluginOnWorkerNodes := hasSourceInMachineConfigStatus(infra.WorkerMCPStatus, infra.WorkerIPsecMachineConfig.Name)
-	return infra.MasterMCPStatus.MachineCount == infra.MasterMCPStatus.ReadyMachineCount &&
-		infra.WorkerMCPStatus.MachineCount == infra.WorkerMCPStatus.ReadyMachineCount &&
-		ipSecPluginOnMasterNodes && ipSecPluginOnWorkerNodes
+	return ipSecPluginOnMasterNodes && ipSecPluginOnWorkerNodes
```

Comment:
But what does this mean for the normal N/S ipsec deployment? 
Is it ok if we rollout the N/S ipsec daemonset before the nodes have actually rebooted after applying the machine configs?
Shouldn't we, instead, track `isNSIPsecActive` and then where we consider `isIpsecMachineConfigActive` use `isIpsecMachineConfigActive || isNSIPsecActive` instead?

Reply:
In the case of N/S IPsec deployment, only ipsec MachineConfigs are deployment with ipsec DaemonSets. so no problem here.
The MachineConfigStatus is updated with IPsec MachineConfig Name once rollout is complete on all nodes. So `isIpsecMachineConfigActive`  becomes true after MC installed on both master and workers nodes. Hence timing of ipsec DS rollout doesn't change.